CruCentralCoast/Events/EventDetailsVC.swift

```diff
@@ -20,10 +20,12 @@ class EventDetailsVC: UIViewController {
     @IBOutlet weak var descriptionLabel: UILabel!
     @IBOutlet weak var closeButton: UIButton!
     @IBOutlet weak var locationButton: UIButton!
+    @IBOutlet weak var facebookButton: CruButton!
     
     @IBOutlet weak var topConstraint: NSLayoutConstraint!
     
     private var currentImageLink: String?
+    private var locationButtonTitle: String = ""
```

Comment:
Why does this variable exist? I don't see it set anywhere

Reply:
You should probably use a local variable to store the concatenated string within the function rather than a class variable. Something like `var locationButtonTitle = self.event?.location?.string ?? ""`

Reply:
I understand now. I had trouble concatenating optional strings, but after unwrapping, it was a lot easier